control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -342,6 +346,16 @@ message SetVolumeReplicaRequest {
   // replica count
   uint32 replicas = 2;
 }
+// Set the volume property
+message SetVolumePropRequest {
+  // uuid of the volume
+  google.protobuf.StringValue uuid = 1;
```

Comment:
@tiagolobocastro @hrudaya21 I think we should create a doc covering protobuf types sometime, otherwise we keep doing same mistakes again and are not consistent.